File: Phan_A/bai37/bai_37.jl
function dem(a)
    chan = 0
    le = 0
    am = 0
    for x in a
        if x % 2 != 0
            le += 1
        else
            chan += 1
        end
        if x < 0
            am += 1
        end
    end
    return chan, le, am
end

input_data = split(read(stdin, String))
if !isempty(input_data)
    n = parse(Int, input_data[1])
    a = [parse(Int, input_data[i]) for i in 2:(n+1)]
    chan, le, am = dem(a)
    println("chan $chan, le $le, am $am")
end

File: Phan_A/bai37/bai_37.py
def dem(a):
    chan = 0
    le = 0
    am = 0
    for x in a:
        if x % 2 != 0:
            le += 1
        else:
            chan += 1
        if x < 0:
            am += 1
    return chan, le, am

if __name__ == "__main__":
    import sys
    input_data = sys.stdin.read().split()
    if input_data:
        n = int(input_data[0])
        a = [int(x) for x in input_data[1:n+1]]
        chan, le, am = dem(a)
        print(f"chan {chan}, le {le}, am {am}")